src/train.js

```diff
@@ -26,6 +28,7 @@ class Tuner {
     // Copy-and-pasted from Fathom just to allow solutionCost() to be async.
     // What color is your function?
     async anneal(updateProgress) {
+        console.time('Training');
```

Comment:
Did you mean to leave these in?

Reply:
yes; we have other console logs in the same neck of the woods, and i figure knowing the run timings may come in useful for debugging at some point.

Reply:
Cool. If the noise bugs us, we can rip it out then.